src/governance.rst

```diff
@@ -0,0 +1,172 @@
+  Each member of core is entitled to one vote on all elected matters.
+* **staged-recipes:** The staged-recipes team administers the staged-recipes
+  repository and is responsible for the review and creation of new feedstocks.
+* **maintainers:** A maintainer is an individual responsible for the management
+  of one or more feedstock repositories and packages. Maintainers have the ability
+  to merge pull requests into the feedstock of only the packages they maintain.
+* **external contributors:** This group encompasses all others who are not on
+  core, part of staged-recipes, or maintainers. This includes first time
+  contributors, collaborators, and funders. They have no special rights within
+  the conda-forge organization itself.
+
+Sub-Teams
+---------
+The core team may elect to create new sub-teams for managing the daily business
+of the organization. While sub-teams may have non-core members, every sub-team
+must have at least one core team member at all times. If a sub-team fails to
+have  a core team member for more than 1 week, that team is considered to be
+dissolved. A new sub-team would need to be established by the core in order to
+reinstate the activity.
+
+Sub-teams have a charter that is either *dynamic* or *static*.
+
+* A *dynamic* charter means that the sub-team is self-organizing, with respect
+  to its own internal policies, procedures, and membership. A sub-team may choose
+  to modify its membership independent of the core team. For example, a
+  Google Summer of Code team could be a good candidate for a dynamic charter.
+  Alternatively, language-based maintenance teams also have a dynamic charter.
+* A *static* charter means that all membership decisions and non-trivial policies
+  changes must be approved by the core team. For example, a finance team
+  may require a static charter.
+
+All sub-teams must adhere to the governance, policies, and procedures of
+conda-forge at all times.
+
+Voting
+------
+This section presents descriptions and criteria for voting items in the
+conda-forge community. The core team is the only team with voting rights.
+Members of core may also call a vote on any topic. The restrictions on
+calling a vote are as follows:
+
+* There must only be one vote active on a particular item at any time.
+* The act of calling for a vote cannot itself violate the code of
+  conduct. For example, Sam repeatedly called for votes immediately
+  after a previous votes failed to achieve Sam's result. Sam is
+  attempting to bully other members of core into agreeing, and is thus
+  violating the code of conduct.
+
+Voting items are labeled as either **standard** or **sensitive**.
+Standard items are ones where public record and discourse and record is
+preferable. Sensitive voting items are ones where the results of the
+vote should remain private to the voters after the vote has occurred.
+Sensitive votes should take place on
+`the Helios voting system <https://vote.heliosvoting.org/>`_ in order
```

Comment:
What should we use for standard voting? Can we use Helios for everything?

Reply:
I figured we wouldn't want to, if we just wanted to have a vote durring a meeting

Reply:
It is good tech, though, and other votes can certianly choose to use it.

Reply:
+1 's on an issue on gh (Apache-ish style?), maybe

Reply:
Do also want to add something to deal with a minimal number of votes?

Reply:
A lack of "no" on the conda-forge dev call today resulted in an implicit "yes" to helios from everyone on the call @CJ-Wright @jakirkham @mariusvniekerk @jjhelmus @msarahan @ocefpaf @scopatz and "justin"